docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md

```diff
@@ -129,77 +133,81 @@ You can create an instance of Yugabyte Platform via he OpenShift web console as
 
 - Open the OCP web console and navigate to **Home > Projects > Create Project**.
 - Enter the name yb-platform and click **Create**.
-- Navigate to **Operators > Installed Operators** select **Yugabyte Platform Operator**, as shown in the following illustration:
+- Navigate to **Operators > Installed Operators** and select **Yugabyte Platform Operator**, as shown in the following illustration:
 
 ![Yugabyte Platform Install Operator](/images/ee/openshift-install-yp-operator.png)
 
 - Click **Create Instance** to open the **Create YBPlatform** page. 
-- Review the default settings without modifying them and click **Create**. Ensure that the **yb-platform** project is selected.
 
-{{< note title="A note on StorageClass" >}}
-If your cluster has a different StorageClass than `standard`, then do the following when you are at the **Create YBPlatform** page:
-- Go to **YAML View**.
-- Change the value of `spec.yugaware.storageClass` to correct StorageClass name.
+- Ensure that the **yb-platform** project is selected and review the default settings.
 
-You can find the StorageClass by going to **Storage > Storage Classes** on the OpenShift Web Console using admin user.
-{{< /note >}}
+- Accept the default settings without modifying them, unless your cluster has a StorageClass other than standard, in which case open **YAML View** and change the value of `spec.yugaware.storageClass` to the correct StorageClass name.
+
+  You can find the StorageClass by navigating to **Storage > Storage Classes** on the OpenShift Web Console as admin user.
```

Comment:
The image is missing here? Also the file name has a typo `openshirt-storageclass.png` should be f instead of r.

Reply:
I thought the image would show a web UI screen. But since it's not, I didn't see any value in adding it.

Reply:
It is a web UI screen only.